src/Functions/GatherUtils/sliceHasImplAnyAll.h

```diff
@@ -419,7 +419,7 @@ DECLARE_SSE42_SPECIFIC_CODE (
 // SSE4.2 Int64, UInt64 specialization
 template<typename IntType>
 requires (std::is_same_v<IntType, Int64> || std::is_same_v<IntType, UInt64>)
-inline ALWAYS_INLINE bool sliceHasImplAnyAllImplInt64(
+inline bool sliceHasImplAnyAllImplInt64(
```

Comment:
why these changes are needed?

Reply:
Explained in commit msg header [here](https://github.com/ClickHouse/ClickHouse/pull/40999/commits/d054ffd1109ab93ec366a64558c7c983cd23a11e).

Reply:
Isn't this code only used under `USE_MULTITARGET_CODE`? I think it'd make more sense to keep the inlines and disable `ENABLE_MULTITARGET_CODE` for this kind of build, right?

Reply:
Yeah, that's the other option I thought about.  My feeling is that removing `FORCE_INLINE` is the cleaner approach. It avoids leaking a instruction set requirement from the optimized code into the generic code. But I also see that quite a few queries slowed down 😞 
Okay, will disable `ENABLE_MULTITARGET_CODE` then ...